tutorial/set-up-stream-client.md

```diff
@@ -1,195 +0,0 @@
-(tut-set-up-stream-client)=
```

Comment:
Are we removing the stream client tutorial because we now want users to use only the dashboard with the appliance?

Also slightly unrelated, will there be a scenario where the stream client set up can be offered for the regular deployment? If yes, I will later redo the content to suit that scenario where users set up a stream client of their choice for regular Anbox Cloud.

Reply:
We the current design of the new appliance the tutorial no longer applies as-is. I tried to make it work but we will require a few changes to make that possible. To not keep not-working content I think it is best to drop the tutorial and bring it back in a next step.

Reply:
Thanks, I will add a Jira ticket to keep track of this.